distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -230,6 +230,28 @@ public void testBasicRestAccess() throws Exception {
     }
 
     @Test
+    public void testSamlFederatedAuth() throws Exception {
+        String recordId = TestCatalog.ingest(Library.getSimpleGeoJson(), "application/json");
+        configureRestForBasic();
+
+        // Creating a new OpenSearch source with no username/password.
+        // When an OpenSearch source attempts to authenticate without a username/password it will
+        // use the subject in the request to create a SAML authentication token
+        OpenSearchSourceProperties openSearchProperties = new OpenSearchSourceProperties(
+                OPENSEARCH_SOURCE_ID);
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchProperties);
+
+        waitForFederatedSource(OPENSEARCH_SOURCE_ID);
+
+        String openSearchQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + OPENSEARCH_SOURCE_ID;
+        given().auth().basic("admin", "admin").when().get(openSearchQuery).then().log().all()
+                .assertThat().statusCode(equalTo(200)).assertThat().body(containsString("myTitle"));
+
+        configureRestForAnonymous();
```

Comment:
configureRestForAnonymous is already in @Before so that line is not needed.  We could use @After to remove all metacards since we do not ingest in the @Before.